src/components/VLanguage/VLanguage.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <VSelectField
+    v-bind="$attrs"
```

Comment:
Is this a leftover of some previous changes? if we remove `inheritAttrs: false`, the $attrs will be bound automatically?

Reply:
Yeah, it's broken.
<img width="743" alt="Screenshot 2022-09-21 at 9 51 58 AM" src="https://user-images.githubusercontent.com/16580576/191424747-708e36f8-144c-445e-83a5-19c3b73b4264.png">